server/session/handler_anvil.go

```diff
@@ -215,9 +215,9 @@ func repairWithDurable(input item.Stack, durable item.Stack, result item.Stack)
 	return result, cost
 }
 
-// mergeEnchantments merges the enchantments of the material item stack onto the result item stack and returns the result
+// anvilMergeEnchantments merges the enchantments of the material item stack onto the result item stack and returns the result
 // item stack, booleans indicating whether the enchantments had any compatible or incompatible enchantments, and the cost.
-func mergeEnchantments(input item.Stack, material item.Stack, result item.Stack, cost int, enchantedBook bool) (item.Stack, bool, bool, int) {
+func anvilMergeEnchantments(input item.Stack, material item.Stack, result item.Stack, cost int, enchantedBook bool) (item.Stack, bool, bool, int) {
```

Comment:
Again, I prefer the old name. Any particular reason to prefix this all with `anvil`?

Reply:
No reason - was a change that I made while testing and forgot to revert.